godtools/App/Preview/Data/RealmDatabase/SwiftUIPreviewDatabaseConfiguration.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  SwiftUIPreviewDatabaseConfiguration.swift
+//  godtools
+//
+//  Created by Levi Eggert on 8/27/23.
+//  Copyright © 2023 Cru. All rights reserved.
+//
+
+import Foundation
+
+class SwiftUIPreviewDatabaseConfiguration: RealmDatabaseConfiguration {
+    
+    init() {
+        
+        super.init(
+            cacheType: .inMemory(identifier: "godtools_swiftui_preview_database"),
```

Comment:
So do previews require the database to be in memory?  I know previews were kind of working at some point, so just curious what changed.

Reply:
Hey @rachaelblue it shouldn't, but there are instances those objects may not exist in the database yet since those are populated on app launch.  

There's a couple other issues going on with previews crashing right now.  One in Firebase Analytics when attempting to track an event.  I also noticed that when the preview is compiled it actually triggers UIApplicationDidBecomeActive in the AppFlow which then causes a crash in the shared parser parse translation manifest async method.